pages/flow.js

```diff
@@ -1,15 +1,180 @@
+  const [reactions, setReactions] = useState([]);
+
+  // state that holds the color that is selected in the first tag cloud
+  const [color, setColor] = useState("grey");
+
+  // state that hold the current page
+  const [page, setPage] = useState(0);
+
+  const guides = [
+    "share your emotions ...",
+    "breathe",
+    "think about your day",
+    "how do you feel?",
+    "what do you feel right now?",
+    "how intense is this feeling for you?",
+    "what's your reaction?",
+  ];
+
+  function PageDisplay() {
+    switch (page) {
+      case 0:
+        return (
+          <Page>
+            <Guide bigger={true} text={"komm zur Ruh"} />
+          </Page>
+        );
+      case 1:
+        return (
+          <Page>
+            <GuideAnimator guides={guides} />
+          </Page>
+        );
+      case 2:
+        return (
+          <Page>
+            <Guide text={guides[4]} />
+            <TagCloud
+              tags={experiences}
+              colorSelected={true}
+              allowMultiple={false}
+              onSelectTag={handleSelectExperience}
+              selectedTags={experience}
+            />
+          </Page>
+        );
+      case 3:
+        return (
+          <Page>
+            <Guide text={guides[5]} />
+            <Slider
+              experience={experience}
+              sliderValue={sliderValue}
+              handleSliderChange={handleSliderChange}
+            />
+          </Page>
+        );
+      case 4:
+        return (
+          <Page>
+            <Guide text={guides[6]} />
+            <TagCloud
+              selectedTags={reactions}
+              tags={experience[0].reactions}
+              colorSelected={true}
+              allowMultiple={true}
+              onSelectTag={handleSelectReactions}
+            />
+          </Page>
+        );
+    }
+  }
+
+  // function handleColorChange(color) {
+  //   setColor(color);
+  // }
+
+  function handleSelectExperience(tags) {
+    setExperience(tags);
+    setColor(tags[0].color);
+  }
+
+  function handleSelectReactions(tags) {
+    setReactions(tags);
+  }
+
+  function handleSliderChange(event) {
+    setSliderValue(event.target.value);
+  }
+
+  function handleSave() {
+    setAllEntries([
+      ...allEntries,
+      {
+        id: nanoid(),
+        experience: experience,
+        reactions: reactions,
+        slider: sliderValue,
+        date: new Date().toLocaleString(),
+      },
+    ]);
+    // router.push("entries");
+    // window.location.reload();
+  }
 
-export default function Flow({ allEntries, handleSubmit, handleRangeChange }) {
   return (
-    <TrackerContainer>
-      <header>
-        <h1>mood tracker</h1>
-      </header>
-      <Form onSubmit={handleSubmit} handleRangeChange={handleRangeChange} />
-      <StyledEntryList allEntries={allEntries} />
-    </TrackerContainer>
+    <>
+      <Animation color={color} opacity={sliderValue} />
+      <FlowContainer>
+        <>{PageDisplay()}</>
```

Comment:
it is fairly uncommon and not recommended to call a Component like a normal function. You should change it to call it like a normal component and also pass it the props it needs and since in every scenario you wrap what you return in a Page component, you can simply write it once here:

```suggestion
        <Page>
          <PageDisplay
            guides={guides}
            experience={experience}
            experiences={experiences}
            page={page}
            reactions={reactions}
            sliderValue={sliderValue}
            handleSliderChange={handleSliderChange}
            handleSelectExperience={handleSelectExperience}
            handleSelectReactions={handleSelectReactions}
          />
        </Page>
```